### ISSUE TITLE
weird error with 0.19.3

### ISSUE THREAD

**Issue by aleqx** [trexminer/T-Rex#109]
0.19.3 seems quite buggy 

Ubuntu 16.04, Nvidia 450.57, Cuda 10.2 

--no-watchdog -o stratum2+tcp://daggerhashimoto.eu.nicehash.com:3353 -u <snip> -p x --api-bind-http 0 --no-nvml -d 0,1,2,3,4,5,6,7,8

```
20201211 23:52:16 WARN: GPU #0(000200): GeForce GTX 1080 Ti, intensity set to 22
20201211 23:52:16 WARN: GPU #1(000400): GeForce GTX 1070, intensity set to 22
20201211 23:52:16 WARN: GPU #2(000500): GeForce GTX 1070, intensity set to 22
20201211 23:52:16 TREX: eyJhIjoiUmRXYk9XWTNlcW1GWDFkZFY2SENEZTJVa0I3WDVCWkZmN1c2dXhQMjI2NzBsZmlzUHFTclByZGVsZDY4ODhldGZJbVRIM2gveEdSbDRaTmFpNnJOTnpMS1NFNGpROW54Q3NvMXR1SkNlSG1RNE1KTE54c0JPaTJRTWxXNFAzM0hGZWF1Wk5CR2pyV3dqT0Q2dG1nMzFKNE1rbTJUejB4VlZtQ3NYRFNaNHRzPSIsImIiOiJRdS92MHk1cXA0NFMrd1llZTFyYllnPT0ifQo=
20201211 23:52:16 WARN: Miner is going to shutdown...
20201211 23:52:16 WARN: GPU #8(000f00): GeForce GTX 1070, intensity set to 22
20201211 23:52:16 WARN: GPU #7(000e00): GeForce GTX 1080 Ti, intensity set to 22
20201211 23:52:16 Main loop finished. Cleaning up resources...
20201211 23:52:16 ApiServer: stopped listening on 0.0.0.0:4068
20201211 23:52:16 WARN: GPU #6(000d00): GeForce GTX 1080 Ti, intensity set to 22
20201211 23:52:17 WARN: GPU #5(000b00): GeForce GTX 1080 Ti, intensity set to 22
20201211 23:52:17 WARN: GPU #4(000a00): GeForce GTX 1080 Ti, intensity set to 22
20201211 23:52:17 WARN: GPU #3(000600): GeForce GTX 1070, intensity set to 22
```

What even is that? How am I supposed to make sense of this? I get that it's base64 of a json object but ...
`20201211 23:52:16 TREX: eyJhIjoiUmRXYk9XWTNlcW1GWDFkZFY2SENEZTJVa0I3WDVCWkZmN1c2dXhQMjI2NzBsZmlzUHFTclByZGVsZDY4ODhldGZJbVRIM2gveEdSbDRaTmFpNnJOTnpMS1NFNGpROW54Q3NvMXR1SkNlSG1RNE1KTE54c0JPaTJRTWxXNFAzM0hGZWF1Wk5CR2pyV3dqT0Q2dG1nMzFKNE1rbTJUejB4VlZtQ3NYRFNaNHRzPSIsImIiOiJRdS92MHk1cXA0NFMrd1llZTFyYllnPT0ifQo=`

0.19.1 works fine ...

**Comment by voyagerft** [comment]
Same error to me too, I had to reinstall version 0.19.1, pending the resolution of the bug

**Comment by trexminer** [comment]
Thanks for reporting, the issue will be fixed in the next version.

**Comment by trexminer** [comment]
Fixed in 0.19.4

**Comment by bchtrue** [comment]
@trexminer the same problem. 0.19.1 working fine, 0.19.3-0.19.5 print same error
20210103 20:29:10 TREX: eyJhIjoieDVxbEJXMWhadjI2TGxWY1pXdFk0RTJNbUxRdDVsZXRhODV5eTl3Rml0Zz0iLCJiIjoiWmRvUzUrT1JPdG05WTVGdlZHSjFLZz09In0K
